src/app/hearing-part/components/draggable-hearing-part/draggable-hearing-part.component.ts

```diff
@@ -0,0 +1,26 @@
+import { AfterViewChecked, ChangeDetectionStrategy, Component, Input} from '@angular/core';
+
+import * as $ from 'jquery';
+import 'jquery-ui/ui/widgets/draggable.js';
+import { HearingPart } from '../../models/hearing-part';
+@Component({
+  selector: 'app-draggable-hearing-part',
+  templateUrl: './draggable-hearing-part.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DraggableHearingPartComponent implements AfterViewChecked {
+    @Input() hearingPart: HearingPart;
+
+    constructor() {
+    }
+
+    ngAfterViewChecked() {
+        ($('.draggable-hearing') as any).draggable({
+            revert: false,
```

Comment:
Shouldn't this be true? What will happen if you drop hearingPart on blank space?

Reply:
Don't worry about it